docs/js/contribution.md

```diff
@@ -0,0 +1,173 @@
+---
+id: contribution-model-sap-cloud-sdk-for-javascript-typescript
+title: Contribution model
+sidebar_label: Contribution model
+description:
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+---
+
+## Introduction
+
+SAP Cloud SDK for JavaScript welcomes contributions that bring value to SDK users by simplifying and/or improving cloud application development within SAP ecosystem. The contributed feature or API library becomes a part of SDK and is released under SAP Cloud SDK umbrella into `@sap-cloud-sdk` to the [npmjs.com](https://www.npmjs.com/).
+
+Most modules of the SAP Cloud SDK for JavaScript are Open Source released under Apache 2.0 license. It's preferred, but not mandatory that contribution happens under same conditions.
+
+## Roles
+
+    - **SDK team** - development team and Product Owner responsible for maintaining, developing and releasing SAP Cloud SDK for JavaScript.
+    - **Contribution team** - development team responsible for developing and contributing a feature or an API library
+
+### Stakeholders and contact points
+
+    - **SDK Team** assigns two main stakeholders as contact points for the contribution process.
+      - **Prodact owner**  for organisational topics, priorities and practicalities
+      - **Developer** for engineering onboarding, alignment and consulting
+    - **Contribution team** assigns at least one contact point with engineering background for regular collaboration and syncs.
+
+Other stakeholders and/or contact points could be assigned if deemed necessary.
+
+## Process outline
+
+Below are generic steps for a successful contribution. The process can be adopted for the needs of specific contribution.
+
+    - Initial alignment and approvals
+    - Plan conribution collaboration
+    - Prepare and configure required assests (repositories, configuration, piplines, etc)
+    - Onboarding session with SDK team. Get to `Hello World!`
+    - Start of development by contribution team
+    - SDK Team provides ongoing consulting
+    - Regular syncs on development and organisational topics
+    - Successful PoC
+    - Iterative development process with SDK team contributing to PR reviews
+    - Releasing the contribution as `Beta`
+    - Iterative development. Stabilizing the API.
+    - Release the contribution as GA aka `General Availability`
```

Comment:
If you do not use GA in the rest of the document is the abbreviation needed? Also I prefer the standard way of defining it `Release the contribution as General Availability (GA)`